vendor/bat-native-ledger/src/bat/ledger/internal/endpoint/gemini/post_account/post_account_gemini.cc

```diff
@@ -13,6 +13,7 @@
 #include "bat/ledger/internal/endpoint/gemini/gemini_utils.h"
 #include "bat/ledger/internal/ledger_impl.h"
 #include "net/http/http_status_code.h"
+#include "third_party/abseil-cpp/absl/types/optional.h"
```

Comment:
Do we need these includes (or if not, are they desirable)? The JSON reader/writer headers would bring it in.

Reply:
I think it's expected to include them according to the IWYU rule that's in effect for Chromium:
https://google.github.io/styleguide/cppguide.html#Include_What_You_Use